site/app/models/User.php

```diff
@@ -611,8 +611,9 @@ public static function validateUserData($field, $data): bool {
                 return preg_match("~^[1-4]{1}$~", $data) === 1;
             case 'registration_section':
                 //Registration section must contain only alpha (upper and lower permitted), numbers, underscores, hyphens.
+                // AND between 0 and 20 chars.
                 //"NULL" registration section should be validated as a datatype, not as a string.
-                return preg_match("~^(?!^null$)[a-z0-9_\-]+$~i", $data) === 1 || is_null($data);
+                return preg_match("~^(?!^null$)[a-zA-Z0-9_\-]{1,20}$~", $data) === 1 || is_null($data);
```

Comment:
I am not sure if the registration section should be case sensitive, which could potentially confuse users.

Reply:
Will discuss about it during the meeting and update the comment.

Reply:
No, it doesn't let me update it.

Reply:
Update: Barb also thinks it should be case-insensitive, doesn't make sense to have Staff section and STAFF section.